bin/win/bootstrap.bat

```diff
@@ -0,0 +1,115 @@
+rem ############################################################################
+rem # Copyright 2016 Intuit
+rem #
+rem # Licensed under the Apache License, Version 2.0 (the "License");
+rem # you may not use this file except in compliance with the License.
+rem # You may obtain a copy of the License at
+rem #
+rem #     http://www.apache.org/licenses/LICENSE-2.0
+rem #
+rem # Unless required by applicable law or agreed to in writing, software
+rem # distributed under the License is distributed on an "AS IS" BASIS,
+rem # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+rem # See the License for the specific language governing permissions and
+rem # limitations under the License.
+rem ############################################################################
+
+
+rem List of dependencies to install
+setlocal
+set choco_packages=docker,docker-machine,jdk8,maven,nodejs.install,ruby,virtualbox --allowEmptyChecksums,git.install
+set npm_packages=bower,grunt-cli,yo
+set gem_packages=compass,fpm
+
+rem Test if administrator (see http://stackoverflow.com/a/11995662)
+net session >nul 2>&1
+if not %errorLevel% == 0 (
+    call :error You need administrator rights to bootstrap Wasabi.
+    endlocal
+    exit /b 1
+)
+
+rem install chocolatey
+call :debug Trying to find Chocolatey
+if not exist C:\ProgramData\chocolatey\choco.exe (
```

Comment:
It's fixed in the latest commit but it seems the reviews don't reflect those changes anymore or something... don't know. If you have time please check it out again and try to run it again.